src/components/atoms/CreatableMultiSelect/CustomComponents/LabelsMultiValueLabel.tsx

```diff
@@ -2,34 +2,18 @@ import {MultiValueGenericProps} from 'react-select';
 
 import {SplitLabelText} from '@atoms';
 
-import {Text} from '@custom-antd';
-
 import {Option} from '@models/form';
 
 import {StyledMultiLabel} from '../CreatableMultiSelect.styled';
 
 const LabelsMultiValueLabel = (props: MultiValueGenericProps<Option>) => {
   const {children} = props;
 
-  const renderChildren = () => {
-    if (typeof children !== 'string') {
-      return null;
-    }
-
-    const trimmedChildren = children.trim();
-
-    if (trimmedChildren.match(/(.+:.+)/g)) {
-      return <SplitLabelText value={trimmedChildren} textClassName="regular small" />;
-    }
-
-    if (trimmedChildren.slice(-1) === ':') {
-      return <Text>{trimmedChildren.slice(0, -1)}</Text>;
-    }
-
-    return <Text>{trimmedChildren}</Text>;
-  };
-
-  return <StyledMultiLabel data-test={`selected-label-${children}`}>{renderChildren()}</StyledMultiLabel>;
+  return (
+    <StyledMultiLabel data-test={`selected-label-${children}`}>
```

Comment:
Earlier it was rendering different components depending on the case (`<Text>`, `<SplitLabelText>`, `<StyledMultiLabel>`), now it's not. Is it expected?